vm/hv1/hv1_emulator/src/synic.rs

```diff
@@ -322,6 +328,11 @@ impl ProcessorSynic {
         self.timers[n].count
     }
 
+    /// Returns the value of the VINA register.
+    pub fn vina(&self) -> HvRegisterVsmVina {
+        self.vina
```

Comment:
It seems like this never changes (I guess this change isn't done)

Reply:
Correct, that's coming soon